flutter/lib/default_integrations.dart

```diff
@@ -84,6 +84,65 @@ Integration runZonedGuardedIntegration(
   return integration;
 }
 
+Integration loadContextsIntegration(
+  SentryOptions options,
+  MethodChannel channel,
+) {
+  Future<void> integration(Hub hub, SentryOptions options) async {
+    try {
+      final Map<String, dynamic> infos =
+          Map<String, dynamic>.from(await channel.invokeMethod('loadContexts'));
```

Comment:
this means we'd `loadContexts` only once, on App. start, wondering if this should not be inside of the `eventProcessor`, so we query up to date data on every event, how is in RN?

Reply:
✅ called for each event